configuration/README.md

```diff
@@ -234,25 +234,6 @@ The supported configuration options:
 
 Audit events are delivered in batched mode, multiple events in one webhook `POST` request.
 
-### `authentication`
```

Comment:
Maybe leave this here as with `network`?

Reply:
done